config_manager.py

```
@@ -0,0 +1,95 @@
+    return False
+
+
+# Mutable default argument
+def add_item(item, items=[]):
+    items.append(item)
+    return items
+
+
+# Wildcard import simulation - polluting namespace
+globals()["DEBUG"] = True
+globals()["SECRET"] = "not-so-secret"
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "SECRET"

(S105)